oio/cli/admin/service_check.py

```diff
@@ -255,3 +257,64 @@ def take_action(self, parsed_args):
         self.logger.info("All rdir services are alive.")
 
         return ('Status', ), [('Ok', )]
+
+
+class RawxCheck(ItemCheckCommand):
+    """
+    Check all rawx chunks
+    """
+
+    columns = ('Chunk', 'Status', 'Errors')
+
+    @property
+    def rdir(self):
+        """Get an instance of RdirClient."""
+        return self.app.client_manager.rdir
+
+    reqid_prefix = 'ACLI-RC-'
+
+    @property
+    def formatter_default(self):
+        return 'value'
+
+    def _format_results(self, checker):
+        for res in checker.run():
+            if res.target.type == "chunk":
+                yield repr(res.target)[len('chunk='):], \
+                    res.health,\
+                    res.errors_to_str()
+
+    def get_parser(self, prog_name):
+        parser = super(RawxCheck, self).get_parser(prog_name)
+        parser.add_argument(
+            'service',
+            nargs='+',
+            metavar='<service_id>',
+            help=('Rawx service to check')
+        )
+        return parser
+
+    def _prefix_http(self, url):
+        if not url.startswith('http://'):
+            return "http://" + url
+
+    def check_services(self, parsed_args):
+        reqid = self.app.request_id(self.reqid_prefix)
```

Comment:
I wanted to use the same request ID to regroup the check to one `action`.
This may be discussed, wanting a different request id for each service check.